articles/tokens/index.md

```diff
@@ -49,7 +49,7 @@ The Refresh token is a long-lived token that is used to obtain a new [`id_token`
 
 ## Delegation Tokens
 
-The Auth0 [`id_token`](#auth0-idtoken-jwt)'s can be exchanged for another token, called a Delegation Token, that can be used to call either other application APIs registered as clients in the same Auth0 tenant or APIs represented by some types of application Addons registered in the same Auth0 tenant.
+The Auth0 [`id_token`](#id-token)'s can be exchanged for another token, called a Delegation Token, that can be used to call either other application APIs registered as clients in the same Auth0 tenant or APIs represented by some types of application Addons registered in the same Auth0 tenant.
```

Comment:
Anything related to delegation is a legacy doc. It doesn't make sense to use cross-client nor third-party delegation on the new pipeline.